src/shared/colors.css

```diff
@@ -53,4 +53,65 @@
 
   /* Notify */
   --rvr-color-notify: var(--rvr-hot-pink);
+
+  /* colors from zeplin styleguide, created by Egle */
+  /* stylelint-disable comment-empty-line-before */
+
+  /* --teal: #00837e; */
```

Comment:
This is from @andrewBalekha's initial pill commit, which I rebased with my changes.

I know we need to re-map the main color variables so we're not hosting both pre and post-rebranding colors.
But, since I don't have the mapping handy, I thought I'd leave the data in here until next time.